catalogs/catalog-hive/src/main/java/com/datastrato/gravitino/catalog/hive/HiveCatalogPropertiesMeta.java

```diff
@@ -60,6 +68,22 @@ public class HiveCatalogPropertiesMeta extends BaseCatalogPropertiesMetadata {
                   DEFAULT_IMPERSONATION_ENABLE,
                   false,
                   false))
+          .put(
+              KET_TAB_URI,
+              PropertyEntry.stringImmutablePropertyEntry(
+                  KET_TAB_URI, "The uri of key tab for the catalog", false, null, false, false))
+          .put(
+              PRINCIPAL,
+              PropertyEntry.stringImmutablePropertyEntry(
+                  PRINCIPAL, "The principal for the catalog", false, null, false, false))
+          .put(
+              REFRESH_INTERVAL_SEC,
+              PropertyEntry.integerOptionalPropertyEntry(
+                  REFRESH_INTERVAL_SEC, "The interval to refresh the principal", true, 60, false))
```

Comment:
60 seconds is too frequent for key refreshment.

Reply:
This interval just check whether the UserGroupInformation is valid. It won't login every time. I will optimize the name and description.